src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -832,6 +832,7 @@ private static int IndexOfNonWhiteSpaceCharacter(ITextSnapshotLine line)
                         {
                             FinishExecute(executionResult.IsSuccessful);
                         }
+                        _window.SubmissionCounter++;
```

Comment:
It doesn't count those commands executed through UI. It also doesn't count any commands executed through keyboard shortcuts.

Will fix for `Reset` and `Clear Screen` UI command.